core/src/main/java/io/grpc/ServerCall.java

```diff
@@ -167,4 +167,12 @@ public boolean isReady() {
    * <p>This method may safely be called concurrently from multiple threads.
    */
   public abstract boolean isCancelled();
+
+  /**
+   * Enables per-message compression, if an encoding type has been negotiated.  If no message
```

Comment:
It seems this implies `setMessageCompression(true)` must be called for messages to be compressed, but isn't it reversed: this is what you should use to disable compression for a particular message?